marketplace-core/pom.xml

```diff
@@ -97,6 +97,10 @@
       <groupId>org.webjars.bower</groupId>
       <artifactId>angular-ui-bootstrap-bower</artifactId>
     </dependency>
+    <dependency>
```

Comment:
Maybe a version that isn't 10 years old?
https://mvnrepository.com/artifact/org.apache.httpcomponents/httpclient
I see this already got approved and is in big data plugin:
https://github.com/pentaho/big-data-plugin/blob/56f061ec426ad7e31e5985d549edd8bad2988d69/assemblies/legacy-plugin/pom.xml#L112